tests/stdlib/tos.nim

```diff
@@ -511,10 +512,14 @@ block ospaths:
   doAssert relativePath("/Users/me/bar/z.nim", "/Users/me", '/') == "bar/z.nim"
   doAssert relativePath("", "/users/moo", '/') == ""
   doAssert relativePath("foo", "", '/') == "foo"
-  doAssert relativePath("/foo", "/Foo", '/') == (when FileSystemCaseSensitive: "../foo" else: ".")
-  doAssert relativePath("/Foo", "/foo", '/') == (when FileSystemCaseSensitive: "../Foo" else: ".")
-  doAssert relativePath("/foo", "/fOO", '/') == (when FileSystemCaseSensitive: "../foo" else: ".")
-  doAssert relativePath("/foO", "/foo", '/') == (when FileSystemCaseSensitive: "../foO" else: ".")
+  doAssert relativePath("/foo", "/Foo", '/') == (
```

Comment:
avoid controversial style changes in unrelated sections of code, some people find the previous style more readable

Reply:
Ah, okay, good to know. I ran `nimpretty` to try to make sure I didn't change the look of anything because I assumed it was always run/it was standard. I'll revert that part